Makefile

```diff
@@ -85,8 +87,8 @@ check-docforge: $(DOCFORGE)
 	@$(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/check-docforge.sh $(REPO_ROOT) $(REPO_ROOT)/.docforge/manifest.yaml ".docforge/;docs/" "gardener-extension-shoot-oidc-service" false
 
 .PHONY: check
-check: $(GOIMPORTS) $(HELM)
-	go vet ./...
```

Comment:
Do we need to remove this check? Is this covered by other linters?

Reply:
There is a default linter ```govet``` which runs go vet https://golangci-lint.run/usage/linters/

Reply:
Thanks for the explanation!